VideoLocker/src/test/java/org/edx/mobile/discussions/DiscussionTextUtilsTest.java

```diff
@@ -0,0 +1,130 @@
+package org.edx.mobile.discussions;
+
+import android.text.SpannableString;
+import android.text.TextUtils;
+import android.text.style.ClickableSpan;
+import android.widget.TextView;
+
+import org.edx.mobile.discussion.DiscussionTextUtils;
+import org.edx.mobile.discussion.IAuthorData;
+import org.edx.mobile.test.BaseTestCase;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Date;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertNotNull;
+import static junit.framework.Assert.assertTrue;
+
+/**
+ * Created by miankhalid on 25/04/2016.
+ */
```

Comment:
Yup! Preferences -> Editor -> File and Code Templates -> Class -> Remove the `#parse("File Header.java")` from the template. Note that the `File Header.java` file is a built-in templating snippet which can be found in the 'Includes' tab. Unfortunately, we don't seem to be checking the project settings under version control at the moment, so this will have to be done by each individual collaborator. Maybe we should include it?